Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -210,6 +210,27 @@ private function getSelectedDebitBrand($brand)
         return null;
     }
 
+    /**
+     * @param string $item
+     * @return string
+     */
+    protected function retrieveImageUrl($item): string
+    {
+        $objectManager =\Magento\Framework\App\ObjectManager::getInstance();
+        $helperImport = $objectManager->get('\Magento\Catalog\Helper\Image');
```

Comment:
@daanvinken please avoid use of the Object Manager and add `Magento\Catalog\Helper\Image` as a dependency to the constructor instead.
Also, please update the description of this PR since it is no longer only about removing commas

Reply:
Will do. Apologies I was accidentally working in the same branch while pushing changes for Klarna. 
I scheduled some time Thursday 11-11 to clean this up.